src/signals/incident-management/containers/FilterTagList/__tests__/FilterTagList.test.js

```diff
@@ -126,7 +110,28 @@ describe('signals/incident-management/containers/FilterTagList', () => {
         ),
       );
 
-      expect(container.querySelectorAll('span')).toHaveLength(6);
+      expect(queryByText('Normaal')).toBeTruthy();
+      expect(queryByText('Tevreden')).toBeTruthy();
+      expect(queryByText('februariplein 1')).toBeTruthy();
+      expect(queryByText('Centrum')).toBeTruthy();
+      expect(queryByText('Westpoort')).toBeTruthy();
+      expect(queryByText('Telefoon – Adoptant')).toBeTruthy();
+      expect(queryByText('Telefoon – ASC')).toBeTruthy();
+
+      expect(container.querySelectorAll('span')).toHaveLength(10);
```

Comment:
Why would you check for `<span>` elements to be in the document? Wouldn't it make more sense to look for specific test ids?

Reply:
gedaan